_services/fractional-coo.html

```diff
@@ -0,0 +1,19 @@
+---
+layout: services
+name: fractional-coo
+title: Fractional COO
+icon: "/img/Fractional.png"
+excerpt: "Reduce startup costs or maintain continuity by hiring a fractional executive. Whether you need to improve processes, manage specific projects, or oversee daily operations, we bring the expertise to ensure your business runs smoothly and efficiently."
+---
+
+<section>
+  <p>
+    Integer rutrum ante et nunc venenatis, id ultricies risus ultricies.
+    Maecenas euismod sapien eu arcu convallis, vitae vestibulum ipsum maximus.
+    Donec eu felis at libero consequat sagittis a et urna. Praesent nec orci at
+    nulla consequat congue ut non arcu. Praesent nec orci at. Bridge the gap
+    between appointments or reduce your costs by hiring a fractional executive.
+    We bring our expertise with us whether you’re looking to make process
+    improvements, manage specific projects or oversee daily operations.
+  </p>
+</section>
```

Comment:
How long do you envision services pages being? What content do you think we should have on there?

We have a mindmap of different "services" or similar we could add to the TODO pile, and I'm happy to write some up... but I think we need to talk about how we pitch a service well :smile:

Reply:
I can try and write a pitch of Business Central for a page if you would like it? Then we can talk about my considerations and compare/contrast with Simon's takes, to land on some general principles for what goes on these pages?

Reply:
See content added to revops services in PR https://github.com/derangeddk/deranged.dk/pull/100

Reply:
Content related to DevOps and other services offered by deranged is more than welcome 🙂